tests/test_simple_ddl_parser.py

```diff
@@ -275,164 +275,6 @@ def test_parser_multiple_tables():
     assert DDLParser(ddl).run() == expected
 
 
-def test_unique_statement_in_columns():
```

Comment:
i moved the tests specific to `UNIQUE` statements into a new file.  If you dislike I can put them back here.